features/step_definitions/images/jenkins.rb

```diff
@@ -102,11 +103,16 @@
 #   non-SSO login.
 # Installation step also installs based on this.
 Given /^I log in to jenkins$/ do
-  if env.version_ge("3.4", user: user) && user.password?
+  if env.version_ge("3.4", user: user) && env.version_le("3.11", user: user) && user.password?
```

Comment:
in this branch we should only support 4.x, conditions for older cluster verisions better be gone. WDYT?

Reply:
not sure what CVP is. In any case, master branch not suitable to run stuff on 3.x

Reply:
CVP is [container verification pipeline tool][https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CVP&title=Container+Verification+Pipeline+E2E+Documentation#ContainerVerificationPipelineE2EDocumentation-ProductTests) which  anli is taking development.
I will remove the conditions for older cluster verisions later.